tests/email_framework/test_email_client_compatibility.py

```
@@ -0,0 +1,293 @@
+        """
+        
+        # Check for proper table attributes
+        assert 'width="100%"' in table_layout
+        assert 'cellpadding="0"' in table_layout
+        assert 'cellspacing="0"' in table_layout
+        assert 'align="center"' in table_layout
+        assert 'width="600"' in table_layout  # Standard email width
+    
+    def test_line_height_consistency(self):
+        """Test line height for readability."""
+        styles_with_line_height = [
```

Ruff (0.12.2)

Local variable `styles_with_line_height` is assigned to but never used

Remove assignment to unused variable `styles_with_line_height`

(F841)